addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -279,8 +279,7 @@
                 </a>
                 <div t-if="not channel.is_member and channel.enroll == 'invite'" class="text-center">
                     <div class="alert my-0 bg-100 p-2">Private Course</div>
-                    <div t-if="channel.enroll_msg" t-field="channel.enroll_msg"/>
-                    <small t-if="not channel.enroll_msg">Contact website administrator.</small>
+                    <div t-field="channel.enroll_msg"/>
```

Comment:
I would add a class on the div, something like "o_wslides_enroll_msg" and "mt-2".

And add some styling there:

```
.o_wslides_enroll_msg {
    p {
        display: inline-block;
        margin-bottom: 0px;
    }
}
```

It will add some spacing on top of the element and also prevent the `<p>`, which looks added by default, to add a lot of margin at the bottom.